libecole/tests/src/reward/test-lpiterations.cpp

```diff
@@ -19,18 +19,16 @@ TEST_CASE("LpIterations return the difference in LP iterations between two state
 	auto model = get_solving_model();
 	reward_func.reset(model);
 
-	SECTION("LP iterations are positive") { REQUIRE(reward_func.obtain_reward(model) > 0); }
+	SECTION("LP iterations are positive") { REQUIRE(reward_func.obtain_reward(model) >= 0); }
 
 	SECTION("LP iterations is zero if no solving happended between two states") {
 		reward_func.obtain_reward(model);
 		REQUIRE(reward_func.obtain_reward(model) == 0);
 	}
 
 	SECTION("Reset LP iterations coutner") {
-		reward_func.obtain_reward(model);
-		REQUIRE(reward_func.obtain_reward(model) == 0);
 		reward_func.reset(model);
-		REQUIRE(reward_func.obtain_reward(model) > 0);
+		REQUIRE(reward_func.obtain_reward(model) >= 0);
```

Comment:
This effectively becomes the same test as the first one. It no longer test that the LP iteration counter is reset.
It just test that `reset` runs smoothly.

Reply:
I though all SECTION tests were run in sequence, and therefore that lines 30-31 were redundant with what was done in the previous test. I'll revert back.